shared/common-adapters/avatar.tsx

```diff
@@ -89,55 +84,31 @@ const followIconHelper = (size: number, followsYou: boolean, following: boolean)
   }
 }
 
-const mapStateToProps = (state: Container.TypedState, ownProps: OwnProps) => {
-  return {
+const ConnectedAvatar = Container.connect(
+  (state, ownProps: OwnProps) => ({
     _counter: state.config.avatarRefreshCounter.get(ownProps.username || ownProps.teamname || '', 0),
     _following: ownProps.showFollowingStatus ? state.config.following.has(ownProps.username || '') : false,
     _followsYou: ownProps.showFollowingStatus ? state.config.followers.has(ownProps.username || '') : false,
     _httpSrvAddress: state.config.httpSrvAddress,
     _httpSrvToken: state.config.httpSrvToken,
-  }
-}
-
-const mapDispatchToProps = (dispatch: Container.TypedDispatch, ownProps: OwnProps) => {
-  return {
-    _goToProfile: (username: string, desktopDest: 'profile' | 'tracker') =>
-      Styles.isMobile || desktopDest === 'profile'
-        ? dispatch(ProfileGen.createShowUserProfile({username}))
-        : dispatch(Tracker2Gen.createShowUser({asTracker: true, username})),
+  }),
+  (dispatch, ownProps: OwnProps) => ({
+    _goToProfile: (username: string) => dispatch(ProfileGen.createShowUserProfile({username})),
     onClick: ownProps.onEditAvatarClick ? ownProps.onEditAvatarClick : ownProps.onClick,
-  }
-}
-
-const avatarSizes = [960, 256, 192]
-const ConnectedAvatar = Container.connect(
-  mapStateToProps,
-  mapDispatchToProps,
+  }),
   (stateProps, dispatchProps, ownProps: OwnProps) => {
+    const {username} = ownProps
     const isTeam = ownProps.isTeam || !!ownProps.teamname
-
-    let onClick = dispatchProps.onClick
-    if (!onClick && ownProps.clickToProfile && ownProps.username) {
-      const u = ownProps.username
-      const desktopDest = ownProps.clickToProfile
-      onClick = () => dispatchProps._goToProfile(u, desktopDest)
-    }
-
-    const style: Styles.StylesCrossPlatform = Styles.isMobile
-      ? ownProps.style
-      : Styles.collapseStyles([
-          ownProps.style,
-          onClick && Styles.platformStyles({isElectron: Styles.desktopStyles.clickable}),
-        ])
-
-    const name = isTeam ? ownProps.teamname : ownProps.username
-    const urlMap = avatarSizes.reduce((m, size: number) => {
+    const onClick =
+      dispatchProps.onClick || (username ? () => dispatchProps._goToProfile(username) : undefined)
```

Comment:
this makes more things clickable, which seems fine to me (cc: @cecileboucheron )